### PR TITLE
Blog about annotations and attributes

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: source/blog/2022-10-26-annotations-to-attributes.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
+---
+title: "From annotations to attributes"
+authorName: Grégoire Paris
+authorEmail: postmaster@greg0ire.fr
+permalink: /2022/10/26/annotations-to-attributes.md
```
Comment (greg0ire) [inline source/blog/2022-10-26-annotations-to-attributes.md:5]: Todo: bump this as well as the filename

**Comment by greg0ire** [comment]
@derrabus you're not in the list of reviewers but I'd like your review :)

**Review by derrabus — APPROVED** [review]
A good read!

File: source/blog/2022-11-04-annotations-to-attributes.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+[annotation-guide]: https://docs.oracle.com/javase/1.5.0/docs/guide/language/annotations.html
+[addendum]: https://github.com/jsuchal/addendum
+[addendum-common]: https://github.com/doctrine/orm/commit/bcf0110249978c8fc705916d5a4f94216bb98b07
+[annotation-driver]: https://github.com/doctrine/orm/commit/9075f10bf5da3058f6ef82ff08e4783ff70424a4
+[annotations-rfc]: https://wiki.php.net/rfc/annotations
+[mailing-list-annotations-rfc]: https://externals.io/message/49733
+
+## Attributes
+
+The RFC mentioned above is only one among [many][rfc]. As mentioned before,
```
Comment (stof) [inline source/blog/2022-11-04-annotations-to-attributes.md:67]: the link reference should be `rfc-list`, not `rfc` (or it should be renamed below)

File: source/blog/2022-11-04-annotations-to-attributes.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+  opcode cache][opcache-save-comments].
+- They are obtained at runtime, by using the reflection API, and because of
+  that, can only be detected as invalid at runtime.
+- They are not well supported by IDEs if at all.
+- They clutter comments, which were originally intended for humans.
+- They can be confused with phpdoc, which are something else entirely.
+
+In March 2020, Benjamin Eberlei resurrects [Dmitry Stogov's attributes
+RFC][attributes-rfc] and submits [the seventh RFC on this
+topic][attributes-rfc], introducing attributes to PHP.
```
Comment (stof) [inline source/blog/2022-11-04-annotations-to-attributes.md:80]: shouldn't this be `attributes-v2-rfc` as link reference ?

File: source/blog/2022-11-04-annotations-to-attributes.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+[rfc-list]: https://wiki.php.net/rfc
+[opcache-save-comments]: https://www.php.net/manual/en/opcache.configuration.php#ini.opcache.save-comments
+[attributes-rfc]: https://wiki.php.net/rfc/attributes
+[attributes-v2-rfc]: https://wiki.php.net/rfc/attributes_v2
+
+## Migrating from one to the other
+
+Since attributes are much better than annotations, with `doctrine/orm` 3.0,
+annotations are no longer supported, which means applications using them as a
```
Comment (stof) [inline source/blog/2022-11-04-annotations-to-attributes.md:95]: should be `will no longer be supported`, as doctrine/orm 3.0 is still a WIP

File: source/blog/2022-11-04-annotations-to-attributes.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+            $rectorConfig->paths([
+                __DIR__ . '/tests',
+            ]);
+            $rectorConfig->sets([
+                DoctrineSetList::ANNOTATIONS_TO_ATTRIBUTES,
+            ]);
+        };
+    ```
+3. Run `vendor/bin/rector`, which obeys the above configuration.
+4. Uninstall Rector: `composer remove rector/rector`
```
Comment (stof) [inline source/blog/2022-11-04-annotations-to-attributes.md:139]: `composer remove rector/rector && rm rector.php` to be exact

**Review by stof — APPROVED** [review]

**Pull request merged**